packages/@webex/plugin-cc/src/services/core/WebSocket/WebSocketManager.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable */
```

Comment:
You could disable the `console.error` -specific check alone
```suggestion
/* eslint no-console: "error" */
```